src/BlazorUI/Bit.BlazorUI/Components/FileUpload/BitFileUpload.ts

```diff
@@ -4,6 +4,47 @@ class BitFileUploader {
     static bitFileUploads: BitFileUpload[];
     static headers: Record<string, string>;
 
+    static initDropZone(dropZoneElement: HTMLInputElement, inputFile: HTMLInputElement) {
```

Comment:
the first argument (dropZonElement) is not of type `HTMLInputElement`. it's `HTMLElement`.